### PR TITLE
Format/refactoring code.

### PR DESCRIPTION
**Pull request by suraj-webkul**
## Issue Reference

Format/refactoring code.

### CODE REVIEW

File: packages/Webkul/Admin/tests/Feature/Catalog/ProductTest.php [hunk 1]
```diff
@@ -148,11 +154,12 @@
 });
 
 it('should perform the mass action forn update status for products', function () {
+    // Arrange
+    $procuts = (new ProductFaker())->getSimpleProductFactory()->count(2)->create();
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/tests/Feature/Catalog/ProductTest.php:158]: products

**Review by devansh-webkul — APPROVED** [review]

File: packages/Webkul/Checkout/src/Database/Factories/CartShippingRateFactory.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    public function definition(): array
+    {
+        $now = date('Y-m-d H:i:s');
+
+        return [
+            'is_calculate_tax'     => 1,
+            'discount_amount'      => 0.0000,
+            'base_discount_amount' => 0.0000,
+            'created_at'           => $now,
+            'updated_at'           => $now,
```
Comment (devansh-webkul) [inline packages/Webkul/Checkout/src/Database/Factories/CartShippingRateFactory.php:29]: now()

File: packages/Webkul/Shop/tests/Feature/Checkout/CheckoutTest.php [hunk 3]
```diff
@@ -0,0 +1,474 @@
...
+    $cartPayment = new CartPayment;
+    $cartPayment->method = $paymentMethod = 'cashondelivery';
+    $cartPayment->method_title = core()->getConfigData('sales.payment_methods.' . $paymentMethod . '.title');
+    $cartPayment->cart_id = $cart->id;
+    $cartPayment->save();
+
+    $cartTemp = new \stdClass();
+    $cartTemp->id = $cart->id;
+
+    session()->put('cart', $cartTemp);
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/tests/Feature/Checkout/CheckoutTest.php:460]: Same here.

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

File: packages/Webkul/Shop/tests/Feature/Checkout/CheckoutTest.php [hunk 4]
```diff
@@ -0,0 +1,474 @@
...
+        ->assertOk()
+        ->assertJsonPath('data.redirect', true)
+        ->assertJsonPath('data.redirect_url', route('shop.checkout.onepage.success'));
+
+    $this->assertDatabaseHas('orders', [
+        'status'          => 'pending',
+        'shipping_method' => 'free_free',
+        'grand_total'     => $price,
+        'cart_id'         => $cart->id,
+    ]);
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/tests/Feature/Checkout/CheckoutTest.php:473]: Check order_items table also.

Comment (devansh-webkul) [reply packages/Webkul/Shop/tests/Feature/Checkout/CheckoutTest.php:473]: Check for order payment and product_ordered_inventory also.

Comment (devansh-webkul) [reply packages/Webkul/Shop/tests/Feature/Checkout/CheckoutTest.php:473]: Check cart table for inactive check.

Comment (devansh-webkul) [reply packages/Webkul/Shop/tests/Feature/Checkout/CheckoutTest.php:473]: Check addresses table for 'order_billing' and 'order_shipping' type.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]
- Check 'order_items' table also.

- Check for 'order_payments' and 'product_ordered_inventory' also.

- Check 'cart' table for inactive cart.

- Check 'addresses' table for 'order_billing' and 'order_shipping' type.

File: packages/Webkul/Shop/tests/Feature/Checkout/CheckoutTest.php [hunk 5]
```diff
@@ -0,0 +1,524 @@
...
+
+    $this->assertDatabaseHas('product_ordered_inventories', [
+        'qty'        => $product->ordered_inventories->pluck('qty')->first(),
+        'product_id' => $product->id,
+    ]);
+
+    $this->assertDatabaseHas('product_inventories', [
+        'product_id' => $product->id,
+        'qty'        => $product->inventories->where('inventory_source_id', 1)->pluck('qty')->first(),
+    ]);
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/tests/Feature/Checkout/CheckoutTest.php:523]: Replace this with `product_inventory_indices` and it should be `quantity-1`.

File: packages/Webkul/Shop/tests/Feature/Checkout/CheckoutTest.php [hunk 6]
```diff
@@ -0,0 +1,524 @@
...
+        'cart_id'      => $cart->id,
+    ]);
+
+    $this->assertDatabaseHas('addresses', [
+        'address_type' => $shippingAddress->address_type,
+        'cart_id'      => $cart->id,
+    ]);
+
+    $this->assertDatabaseHas('order_items', [
+        'qty_ordered'  => 1,
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/tests/Feature/Checkout/CheckoutTest.php:508]: $cartItem->quantity

**Review by devansh-webkul — APPROVED** [review]

**Pull request merged**